docs/operations/karpenter.md

```diff
@@ -2,68 +2,117 @@
 
 [Karpenter](https://karpenter.sh) is a Kubernetes-native capacity manager that directly provisions Nodes and underlying instances based on Pod requirements. On AWS, kOps supports managing an InstanceGroup with either Karpenter or an AWS Auto Scaling Group (ASG).
 
+## Prerequisites
+
+Managed Karpenter requires kOps 1.34+ and that [IAM Roles for Service Accounts (IRSA)](/cluster_spec#service-account-issuer-discovery-and-aws-iam-roles-for-service-accounts-irsa) be enabled for the cluster.
+
 ## Installing
 
-If using kOps 1.26 or older, enable the Karpenter feature flag :
+### New clusters
```

Comment:
As we feature flagged karpenter, I think it is OK to remove support for the old pre 1.0 version of karpenter.  (Users that doesn't want to upgrade to karpenter 1.0+ can stay with the older version of kOps for a while). I think that's a reasonable position for us to take here...

Reply:
Karpenter is no longer feature-flagged...
We removed the flag just before the major changes.